config.default.yaml

```diff
@@ -408,17 +677,68 @@ solving:
     clip_p_max_pu: 0.01
     skip_iterations: true
     track_iterations: false
-    #nhours: 10
+    # nhours: 10
+
   solver:
     name: gurobi
-    threads: 4
-    method: 2 # barrier (=ipm)
-    crossover: 0
-    BarConvTol: 1.e-5
-    FeasibilityTol: 1.e-6
-    AggFill: 0
-    PreDual: 0
-    GURO_PAR_BARDENSETHRESH: 200
+    options: gurobi-default
```

Comment:
I understand this incorporates the pypsa-linopy PR?

Reply:
no it does not, but we need it to for the flexible configuration for testing, but it will be fully compatible with the linopy implementation and thus not lead to (significant) merge conflicts